Gemfile

```diff
@@ -18,4 +18,8 @@ group :jekyll_plugins do
   gem "jekyll-tabs"
 end
 
+group :jekyll_plugins do  
+  gem "jekyll-relative-links"
```

Comment:
You should be able to add this to the block in line 17

Reply:
Done!